vite.config.ts

```diff
@@ -15,6 +17,11 @@ export default defineConfig({
     __BUILD_BRANCH__: JSON.stringify(process.env.VITE_GIT_BRANCH),
     __BUILD_REVISION__: JSON.stringify(process.env.VITE_GIT_REVISION),
   },
+  resolve: {
```

Comment:
Same: Since it is used only 1-5 times, please remove it. Brings unnecessary complexity and inconsistency to the project :)

Reply:
See https://github.com/MORE-Platform/more-studymanager-frontend/pull/585#discussion_r2053591102